### DISCUSSION TITLE
Handy/Fancy Hover for Edges

### DISCUSSION THREAD

**Discussion by AndrewJSchoen** [xyflow/xyflow#2583] [Show and tell]
Hey all,

I made a little implementation of an interactable-on-hover edge that can be seen in this video.
The source code is [here](https://github.com/Wisc-HCI/simple-vp/blob/master/src/components/CanvasEdge.jsx) (see `CanvasEdge`). It uses the `EdgeLabelRenderer` and the `framer-motion` library.

https://user-images.githubusercontent.com/5341396/203417209-6dd79550-e20d-4f71-be4a-aac626ee8888.mov

**Comment by bcakmakoglu** [comment]
That looks nice 👍 good job

**Reply by moklick** [reply]
wow 😳 I love it!
